ai/prompt_context/transform_markdown.py

```diff
@@ -0,0 +1,126 @@
+"""
+This script replaces mkdocs code and api placeholders with the actual code examples and
+API docstrings and writes the modified output to a file.
+
+This code needs to import mesop, so your terminal needs to have mesop imported via pip.
+It does not seem to work when .cli.venv is loaded.
+"""
+
+import argparse
+import os
+
+import mesop as me
+import mesop.labs as mel
+
+DOCS_PATH = "../../docs/"
+MARKDOWN_DOCS_CONTEXT_FILE = "markdown_docs_context.txt"
+MARKDOWN_DOCS_BLOCKLIST_FILE = "markdown_docs_blocklist.txt"
+OUTPUT_DIR = "../gen/extracted_markdown/"
+
+
+def main():
+  parser = argparse.ArgumentParser(
+    prog="Transform Markdown",
+    description="Transform markdown documents for prompt context.",
+  )
+  parser.add_argument("--use_cached_files_list", action="store_true")
+  parser.add_argument("--cache_files_list", action="store_true")
+  args = parser.parse_args()
+
```

Comment:
Add a check to make sure both flags are not set to True? (it seems like it would be a mistake)